pennylane/devices/default_tensor.py

```diff
@@ -238,22 +286,62 @@ def circuit(theta, phi, num_qubits):
             >>> circuit(theta, phi, num_qubits)
             [-0.9953099539219951, 0.0036631029671767208, 0.9999999876072984]
 
-            After the first execution, the time to run this circuit for 50 qubits is around 0.5 seconds depending on the machine.
+            After the first execution, the time to run this circuit for 50 qubits is around 0.5 seconds on a standard laptop.
             Increasing the number of qubits to 500 brings the execution time to approximately 15 seconds, and for 1000 qubits to around 50 seconds.
 
             The time complexity and the accuracy of the results also depend on the chosen keyword arguments for the device, such as the maximum bond dimension.
             The specific structure of the circuit significantly affects how the time complexity and accuracy of the simulation scale with these parameters.
+
+
+            We can also simulate quantum circuits using the Tensor Network (TN) method. This can be particularly useful for circuits that build up entanglement.
+            The following example shows how to execute a quantum circuit with the TN method and configurable depth using ``default.tensor``.
+
+            We set the contraction technique to ``"auto-split-gate"``. With this option, each gate is lazily added to the tensor network
+            and nothing is initially contracted, but the gate is automatically split if this results in a rank reduction.
+
+
+            .. code-block:: python
+
+                import pennylane as qml
+
+                phi = 0.1
+                dept = 10
+                num_qubits = 20
+
+                dev = qml.device("default.tensor", method="tn", contract="auto-split-gate")
+
+                @qml.qnode(dev)
+                def circuit(phi, depth, num_qubits):
+                    for qubit in range(num_qubits):
+                        qml.X(wires=qubit)
+                    for _ in range(depth):
+                        for qubit in range(num_qubits - 1):
+                            qml.CNOT(wires=[qubit, qubit + 1])
+                        for qubit in range(num_qubits):
+                            qml.RX(phi, wires=qubit)
+                    for qubit in range(num_qubits - 1):
+                        qml.CNOT(wires=[qubit, qubit + 1])
+                    return qml.expval(qml.Z(0))
+
+            >>> circuit(phi, dept, num_qubits)
+            -0.9511499466743278
+
+            The execution time for this circuit with the above parameters is around 0.2 seconds on a standard laptop.
+
+            Using ``quimb`` as the backend, the exact tensor network method can be faster than MPS and state vector methods in some cases.
```

Comment:
```suggestion
            The exact tensor network method can be faster than MPS and state vector methods in some cases.
```

Since `quimb` is the only available backend it's okay to leave that I think